README.md

```diff
@@ -22,6 +20,13 @@ availability under heavy load or in the face of resource failure. **AbsaOSS/k3d-
 functionality**. For instance, the E2E use-case is [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) testing
  in [AbsaOSS/k8gb](https://github.com/AbsaOSS/k8gb). 
 
+<p align="center">
+  <a href="https://www.youtube.com/embed/eZcAvTb0rbA" title="Github Actions review and tutorial by #DevOps Toolkit">
+    <img src="https://user-images.githubusercontent.com/7195836/118461760-4a558880-b6fe-11eb-9ffc-5a87c87ed543.png">
```

Comment:
youtube provides a stable location for thumbnails retrieval
```suggestion
    <img src="https://img.youtube.com/vi/eZcAvTb0rbA/0.jpg">
```
We use this way constantly in k8gb readme

Reply:
I see, but I didn't want to use pre-defined thumbnails, that's why I created an image of video:
![](https://user-images.githubusercontent.com/7195836/118461760-4a558880-b6fe-11eb-9ffc-5a87c87ed543.png)

Reply:
ok, didn't realize it was an intentions